src/App.js

```diff
@@ -102,6 +102,24 @@ export default function App() {
         setOutput(translation["response"])
     }
 
+    /**
+     * Function accesses google search custom bar to add query from input 
+     * text box directly into search bar which then searches google. 
+     */
+    function querySearch(inputValue, outputLanguageValue) {
```

Comment:
Nit: Not sure I like the "output" prefix here. Within the context of this function, "output" doesn't make sense.

Reply:
Okay- will add a TODO because I think that otherwise I will get a conflict with another open PR.